example/lib/meeting/meeting_store.dart

```diff
@@ -1295,13 +1327,53 @@ class MeetingStore extends ChangeNotifier
   }
 
   void enterPipMode() async {
```

Comment:
rename function to avoid Android / iOS confusion for readers
```suggestion
  void enterPipModeOnAndroid() async {
```